src/components/manage/Tiles/Rss/Edit.jsx

```diff
@@ -0,0 +1,288 @@
+/**
+ * Edit rss tile.
+ * @module components/manage/Tiles/Rss/Edit
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
+import { Button, Form, Input, Feed, Message, Container,Card, Image } from 'semantic-ui-react';
+import Parser from 'rss-parser';
+import { Icon } from '../../../../components';
+import trashSVG from '../../../../icons/delete.svg';
+import clearSVG from '../../../../icons/clear.svg';
+import imageLeftSVG from '../../../../icons/image-left.svg';
+import imageRightSVG from '../../../../icons/image-right.svg';
+import imageFitSVG from '../../../../icons/image-fit.svg';
+import imageFullSVG from '../../../../icons/image-full.svg';
+import rssSVG from '../../../../icons/rss.svg';
+import { connect } from 'react-redux';
+import { settings } from '~/config';
+import View from './View';
+
+const messages = defineMessages({
+  save: {
+    id: 'Save',
+    defaultMessage: 'Save',
+  },
+  VideoFormDescription: {
+    id: 'Specify a Rss  url',
+    defaultMessage: 'Specify a Rss url',
+  },
+  VideoTileInputPlaceholder: {
+    id: 'Enter site URL',
+    defaultMessage: 'Enter site URL',
+  },
+});
+
+@injectIntl
+/**
+ * Edit rss tile class.
+ * @class Edit
+ * @extends Component
+ */
+class Edit extends Component {
+  /**
+   * Property types.
+   * @property {Object} propTypes Property types.
+   * @static
+   */
+  static propTypes = {
+    selected: PropTypes.bool.isRequired,
+    tile: PropTypes.string.isRequired,
+    data: PropTypes.objectOf(PropTypes.any).isRequired,
+    onChangeTile: PropTypes.func.isRequired,
+    onSelectTile: PropTypes.func.isRequired,
+    onDeleteTile: PropTypes.func.isRequired,
+    intl: intlShape.isRequired,
+  };
+
+  /**
+   * Constructor
+   * @method constructor
+   * @param {Object} props Component properties
+   * @constructs WysiwygEditor
+   */
+  constructor(props) {
+    super(props);
+
+    this.onChangeUrl = this.onChangeUrl.bind(this);
+    this.onSubmitUrl = this.onSubmitUrl.bind(this);
+    this.state = {
+      url: '',
+      feed: {},
+    };
+  }
+
+  /**
+   * Change url handler
+   * @method onChangeUrl
+   * @param {Object} target Target object
+   * @returns {undefined}
+   */
+  onChangeUrl({ target }) {
+    this.setState({
+      url: target.value,
+    });
+  }
+
+  /**
+   * Submit url handler
+   * @method onSubmitUrl
+   * @returns {undefined}
+   */
+  onSubmitUrl(e) {
+    this.props.onChangeTile(this.props.tile, {
+      ...this.props.data,
+      url: this.state.url,
+    });
+    if (this.state.url.match('.rss')){
+    (async () => {
+      const parser = new Parser();
+      const myfeed = await parser.parseURL(this.state.url);
+      this.setState({
+        feed: myfeed,
+      });
+      console.log(myfeed, 'this is of type ' + typeof(this.state.feed.items))
```

Comment:
Ping!

Reply:
ah. crap! sorry, I used it for testing..